internal/policy/verify.go

```diff
@@ -169,6 +169,9 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 			slog.Debug(fmt.Sprintf("Verifying entry '%s'...", entry.ID.String()))
 
 			slog.Debug("Checking if entry is for policy reference...")
+			if entry.RefName == PolicyStagingRef {
```

Comment:
Move this check above the debug statement and add a debug message saying checking if it's for policy-staging?

Reply:
Fixed